hugegraph-pd/pom.xml

```diff
@@ -119,14 +119,6 @@
                             <goal>clean</goal>
                         </goals>
                     </execution>
-                    <!-- auto delete .flattened-pom.xml after "install" step -->
-                    <execution>
-                        <id>remove-flattened-pom</id>
-                        <phase>install</phase>
-                        <goals>
-                            <goal>clean</goal>
-                        </goals>
-                    </execution>
```

Comment:
enhance later: could add an activation (then enable it when not deploying, but only effective during installation).